docs/applying-storybook.md

```diff
@@ -157,3 +158,56 @@ Outlined below are each area going from left to right in the selections.
 | 10. A11y Visualization Simulator         |
 | 11. Set layout direction (left or right) |
 | 12. Toggle color mode                    |
+
+## Chromatic
+
+Chromatic is a visual testing tool that scans every possible UI state across browsers to catch bugs in appearance and functionality. It enables you to assign reviewers and resolve discussions to streamline team sign-off. It is created by the same team that made Storybook. [Read more in the Chromatic docs](https://www.chromatic.com/docs/)
+
+When creating a story, Chromatic creates a "snapshot" of it and sets it as a baseline. This baseline is also approved or denied before merging into the project. Whenever there are changes that affect the component, Chromatic will create a new snapshot to analyze. If there are changes, Chromatic will provide them for a reviewer to accept or decline, and be able to provide any further comments.
+
+Depending on the component, we might look for more than just one snapshot per story. In some cases, we might want multiple snapshots showing the story rendered at various viewport widths or in different languages, a combination of both, etc. These are referred to as [Story Modes](https://www.chromatic.com/docs/modes/). Examples of applicable components include the `Footer` and the `HubHero`.
+
+You will currently find the setup of these modes in [the `./storybook/modes.ts` file](../.storybook/modes.ts)
+
+> Note: At this time we are only considering modes for viewport and languages. Color mode is not possible with the existing setup and is being investigated on making it available, should we want to use it.
+
+When using a mode at either the component level (all stories in a given file) or at the story level, they are supplied under the `chromatic` parameter.
+
+```tsx
+import { Meta, StoryObj } from "@storybook/react"
+
+import { langViewportModes } from "../../../../.storybook/modes"
+
+import ContentHeroComponent, { ContentHeroProps } from "."
+
+const meta = {
+  title: "Organisms / Layouts / Hero",
+  component: ContentHeroComponent,
+  parameters: {
+    chromatic: {
+      modes: {
+        ...langViewportModes,
+      },
+    },
+  },
+  // other options as needed
+} satisfies Meta<typeof ContentHeroComponent>
+```
+
+In this example, we are supplying all the combinations of the languages and viewports together in snapshots. These will only be viewed in chromatic and cannot be seen when viewing storybook locally.
+
+If needs to be only a couple of options, you can write them like this:
+
+```ts
+import { viewportModes, langViewportModes } from "../../../../.storybook/modes"
+
+// In the `meta` object
+parameters: {
+  chromatic: {
+    modes: {
+      base: viewportModes['base']
+      'ru-xl': langViewportModes['ru-xl']
+    }
+  }
+}
+```
```

Comment:
Great job @TylerAPfledderer, I think you mentioned all the important parts. Maybe a nice to have would be a mention of the `disableSnapshot` option since we will probably use it in cases where we want the story but not the visual test.

Reply:
@pettinarip I will review and apply a section covering this. Thank you!

Reply:
@pettinarip this has now been added.